packages/api-clients/open-api/purposeApi.yml

```diff
@@ -415,6 +415,53 @@ paths:
             application/problem+json:
               schema:
                 $ref: "#/components/schemas/Problem"
+    patch:
+      parameters:
+        - $ref: "#/components/parameters/CorrelationIdHeader"
+      tags:
+        - purpose
+      operationId: PATCH_updatePurpose
```

Comment:
Let's remember to track in the API V2 board to remove the POST methods and replace them with these PATCH at the end of the API V2 developments